vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/select_autocomplete.tsx

```diff
@@ -70,7 +70,7 @@ export default function SelectAutocompleteInput(
       clearOnBlur={true}
       options={options.display}
       // disable filtering on client
-      filterOptions={(x: typeof options) => x}
+      filterOptions={(x: string[]) => x}
```

Comment:
Ohhh this must have fallen out-of-date and is remnant from before I would have turned options from a 'string[]' itself into the container of filtered and display 'string[]'s.  :+1: seems right.  The goal right here is to turn off MUI Autocomplete's internal filtering, because we do our own.  Our filtering results get passed in via `options={options.display}` above.